gen_initramfs.sh

```diff
@@ -1998,11 +2130,17 @@ create_initramfs() {
 	append_data 'modprobed'
 	append_data 'multipath' "${MULTIPATH}"
 	append_data 'splash' "${SPLASH}"
+	append_data 'plymouth' "${PLYMOUTH}"
 	append_data 'strace' "${STRACE}"
 	append_data 'unionfs_fuse' "${UNIONFS}"
 	append_data 'xfsprogs' "${XFSPROGS}"
 	append_data 'zfs' "${ZFS}"
 
+	if isTrue "${PLYMOUTH}"
```

Comment:
why not just `append_data 'drm' "${PLYMOUTH}"`?

Reply:
Ah, I got a bit confused by the surrounding "if isTrue" functions.

Reply:
Actually, looking at it again, I'm doing the same thing as ZFS.  Both plymouth and ZFS have an initial append_data() call, then do an "if isTrue" for additional append_data() calls.  Should I not be doing this?